### PR TITLE
fix: daemon command

### LINKED ISSUE #1184: bug: tensorboard not working
## Description
the demo code from web docs is not working.
## Reproduction
```python
def tensorboard(port):
    runtime.mount(host_path="/var/log/tensorboard", envd_path="/home/envd/logs")
    runtime.daemon(commands=[
        ["tensorboard", "--logdir", "/home/envd/logs", "--port", "8888", "--host", "0.0.0.0", ">>tensorboard.log", "2>&1"],
    ])
    runtime.expose(envd_port=port, host_port=port, service="tensorboard")


def build():
    base(os="ubuntu20.04", language="python")
    install.python_packages(name=["tensorboard"])
    tensorboard(8888)
```
## Additional Info
log inside envd container
```shell
⬢ [envd]❯ tail /var/log/horust/daemon_0_stderr.log 
                   [--reload_interval SECONDS] [--reload_task TYPE]
                   [--reload_multifile BOOL]
                   [--reload_multifile_inactive_secs SECONDS]
                   [--generic_data TYPE]
                   [--samples_per_plugin SAMPLES_PER_PLUGIN]
                   [--detect_file_replacement BOOL]
                   [--whatif-use-unsafe-custom-prediction YOUR_CUSTOM_PREDICT_FUNCTION.py]
                   [--whatif-data-dir PATH]
                   {serve,dev} ...
tensorboard: error: unrecognized arguments: >>tensorboard.log 2>&1 &
```
I change the commands as `["tensorboard", "--logdir", "/home/envd/logs", "--port", "8888", "--host", "0.0.0.0", "serve"],`, still get an err

```shell
⬢ [envd]❯ tail /var/log/horust/daemon_0_stderr.log 
                   [--reload_interval SECONDS] [--reload_task TYPE]
                   [--reload_multifile BOOL]
                   [--reload_multifile_inactive_secs SECONDS]
                   [--generic_data TYPE]
                   [--samples_per_plugin SAMPLES_PER_PLUGIN]
                   [--detect_file_replacement BOOL]
                   [--whatif-use-unsafe-custom-prediction YOUR_CUSTOM_PREDICT_FUNCTION.py]
                   [--whatif-data-dir PATH]
                   {serve,dev} ...
tensorboard: error: unrecognized arguments: &
```
```shell
envd: v0.2.4
  BuildDate: 2022-10-31T03:09:28Z
  GitCommit: d693254c1a8b19e02aca85934818a4e7b6dde661
  GitTreeState: clean
  GitTag: v0.2.4
  GoVersion: go1.18.7
  Compiler: gc
  Platform: linux/amd64
```

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

### PR DESCRIPTION
**Pull request by kemingy**
Signed-off-by: Keming <kemingyang@tensorchord.ai>

* fix #1184 

cc @VoVAllen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,8 +52,12 @@ def build():
   <summary><code>envdlib.tensorboard</code> is defined in <a href="https://github.com/tensorchord/envdlib/blob/main/src/monitoring.envd">github.com/tensorchord/envdlib</a></summary>
 
 ```python
-def tensorboard(envd_port=6006, envd_dir="/home/envd/logs",
-        host_port=0, host_dir="/var/log/tensorboard"):
+def tensorboard(
+    envd_port=6006,
+    envd_dir="/home/envd/logs",
+    host_port=0,
+    host_dir="/tmp",
```
Comment (kemingy) [inline README.md:59]: Check https://github.com/tensorchord/envdlib/pull/11

I'm not sure which dir should we use.

Comment (VoVAllen) [reply README.md:59]: Shall we provide function `empty_dir()` like k8s?

Comment (kemingy) [reply README.md:59]: SGTM. Should it create a folder under `.cache/envd`? We can open an issue to track this. I guess `/tmp` works for now. WDYT

cc @gaocegege

Comment (gaocegege) [reply README.md:59]: We can open an issue to keep track. /tmp LGTM for now.

Comment (kemingy) [reply README.md:59]: So can we merge this one?

**Pull request merged**